src/database/utilities/createRecord.js

```diff
@@ -142,7 +142,7 @@ const createRequisition = (
   user,
   { otherStoreName, program, period, orderType = {}, monthsLeadTime = 0 }
 ) => {
-  const { name: orderTypeName, maxMOS, thresholdMOS } = orderType;
+  const { name: orderTypeName = '', maxMOS = 0, thresholdMOS = 0 } = orderType || {};
```

Comment:
Nice way of setting defaults. Do you need the or guard with `orderType = {}` in the parameters?

Reply:
unfortuantely defaults only work with undefined - not null - pretty ugly 🤣

Reply:
😢 .